### ISSUE TITLE
Missing organizations in the organization list:

### ISSUE THREAD

**Issue by vagarwalla** [rtcharity/eahub.org#995]
Add to the Profile organizations in the organization list (these ones seem more obvious):

Future of Humanity Institute
Center on Long-Term Risk
Animal Ethics
Giving What We Can
LessWrong
Center for Applied Ratioanlity
Happier Lives Institute
Innovations for Poverty Action
J-PAL
All-Party Parliamentary Group for Future Generations 
Centre for Enabling EA Learning and Research
The Legal Priorities Project
Sentience Politics
WANBAM

Less clear if we should add them: 
- Do we want a bunch of area-specific giving platforms?
- Do we want CE incubated charities?
- Do we want GiveWell recommended charities?

Basically do we want the list of organisations listed here: https://forum.effectivealtruism.org/posts/f6kg8T2Lp6rDqxWwG/list-of-ea-related-organisations

Or should we do it on a tagging basis later?

@sebgrebe @victor-yunenko

**Comment by vagarwalla** [comment]
Hey guys! Just realised the EA Hub is missing from the organisation list!  ;D
Can we list as: Effective Altruism Hub.

**Comment by sebgrebe** [comment]
> 
> 
> Hey guys! Just realised the EA Hub is missing from the organisation list! ;D
> Can we list as: Effective Altruism Hub.

We're too humble. I guess this makes the Hub becoming an independent entity official? ;)

**Comment by sebgrebe** [comment]
@vagarwalla to answer your question regarding which ones to add: I'd be happy to add the orgs listed in the forum post (you might also have seen that there is a comment explicilty mentioning the Hub, criticising that we don't have those lists).

Generally, this raises the interesting question of how this should be decided. We don't want to play gate keeper, but of course we also shouldn't add orgs willy-nilly. Ideally, we'd have some criteria written up somewhere on how to decide this.

Probably a good topic for our weekly call.

**Comment by victor-yunenko** [comment]
Personally I would just satisfy the majority of users' requests. The organizations are filterable, and there's not much difference in having 100 or 1000 organizations. As long as the end-users are happy, why not?

**Comment by sebgrebe** [comment]
> 
> 
> Personally I would just satisfy the majority of users' requests. The organizations are filterable, and there's not much difference in having 100 or 1000 organizations. As long as the end-users are happy, why not?

The tricky bit is what if the end users disagree? Someone might want to add an org that the vast majority of users wouldn't consider EA. Or someone might want to add an org which is a prototype of a non-EA org (e.g., "Make a wish foundation")

**Comment by victor-yunenko** [comment]
It's a fair point, but before discussing it in detail I would wait until a disagreement occurs since it seems as it might not happen?

**Comment by victor-yunenko** [comment]
For me it would be ok, the field is called "organizational afflictions", not "official EA organizational affiliations" :)

**Comment by vagarwalla** [comment]
- Will add all the organisations listed here: https://forum.effectivealtruism.org/posts/f6kg8T2Lp6rDqxWwG/list-of-ea-related-organisations#comments
Vaidehi has requested access to the Notion in case there are more, but if this issue is not updated by the time someone is working on it you can just use the forum post.

**Comment by vagarwalla** [comment]
EA Hub is mispelt on the org list.

**Comment by victor-yunenko** [comment]
@vagarwalla, can this be closed now?
